ansible_collections/arista/avd/docs/how-to/tower-integration.md

```diff
@@ -99,7 +99,7 @@ $ curl -X PATCH 'http://admin:password@<IP-of-AWX-INSTANCE>/api/v2/settings/syst
 }
 ```
 
-### Provision virtual-environment
+### Provision virtual environment
```

Comment:
The hyphen is more correct since we are talking about the term venv / virtual-env. Not just any virtual environent